packages/frontend-api/src/Model/Order/OrderDataFactory.php

```diff
@@ -98,7 +98,7 @@ public function createOrderDataFromArgument(Argument $argument): OrderData
         }
 
         $orderData->domainId = $this->domain->getId();
-        $orderData->origin = self::ORDER_ORIGIN_GRAPHQL;
+        $orderData->origin = static::ORDER_ORIGIN_GRAPHQL;
```

Comment:
Interesting change, why was this made?

Reply:
While testing individual packages, I notice standards are failing in this package. And for me, it was more like a typo (as we have a rule for this).
Do you want a separate PR?